app/components/provider_interface/individual_condition_status_review_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<span class="govuk-caption-l"><%= application_choice.application_form.full_name %></span>
+<h1 class="govuk-heading-l"><%= page_heading %></h1>
+
+<%= render ProviderInterface::OfferSummaryComponent.new(
+  application_choice: application_choice,
+  course_option: application_choice.current_course_option,
+  conditions: form_object.conditions,
+  course: application_choice.current_course,
+) %>
```

Comment:
There's a bug with this at the moment where if the course has multiple study modes, then a change link renders. Need to modify the base component to fix this

Reply:
Resolved by extending the `editable` attribute of the offer summary component

Reply:
Can the ReadOnlyCompletedOfferSummaryComponent not be used instead?

Reply:
Oh I didn't realise that was a thing! What is the intended difference between `editable: false` and the read only component?

Reply:
I don't remember the details but editable was introduced in order to be used in conjuction with the ReadOnly view. The read only view does something similar to what you've done in OfferSummary, by updating `rows` to not include any links.

https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/views/provider_interface/offers/show.html.erb#L27

Reply:
Ah yes, what I've done won't work. `editable: false` also disables editing of the conditions, which I do need here.

I'll use the readonly one here and maybe we can revisit the components another time if necessary